solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -444,7 +448,7 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
             reserves[reserveToken].balance = newReserveBalance;
 
             // transfer each one of the reserve amounts from the pool to the user
-            if (reserveToken == ETH_RESERVE_ADDRESS)
+            if (address(reserveToken) == ETH_RESERVE_ADDRESS)

```

Comment:
If we use `ETH_RESERVE_ADDRESS` as `IERC20Token` everywhere in this file, then you may want to consider declaring it as such to begin with.